### PR TITLE
jQuery Events

### PR DESCRIPTION
**Pull request by cynthia-cwhq**

### CODE REVIEW

File: docs/javascript/browser-apis-and-jquery.md [hunk 1]
```diff
@@ -446,3 +446,308 @@ function increaseCatLikes() {
...
+        <input id="sub" type="submit" value="Submit">
+    </form>
+
+
+    <script>
+
+    function clickHeading() {
+        $("#heading").show();
+    }
+    // when button is clicked it will show the hidden heading text
```
Comment (daniel-schroeder-dev) [inline docs/javascript/browser-apis-and-jquery.md]: Let's try to be really explicit with things like this.

So:

```javascript
// When the button with the id of `name` is clicked, the `clickHeading()` function will run.
```

File: docs/javascript/browser-apis-and-jquery.md [hunk 2]
```diff
@@ -446,3 +446,308 @@ function increaseCatLikes() {
...
+
+
+    </script>
+
+</div>
+
+</html>
+```
+
+The `dblclick()` method allows us to execute a function when element is double clicked.
```
Comment (daniel-schroeder-dev) [inline docs/javascript/browser-apis-and-jquery.md]: This wasn't in the list of methods to add docs for, and I don't see it in any current courses when I `grep` around the CWHQ projects repo.

File: docs/javascript/browser-apis-and-jquery.md [hunk 3]
```diff
@@ -446,3 +446,308 @@ function increaseCatLikes() {
...
+
+
+    </script>
+
+</div>
+
+</html>
+```
+
+The `hover()` method allows us to execute a function when element is hovered.
```
Comment (daniel-schroeder-dev) [inline docs/javascript/browser-apis-and-jquery.md:502]: I could swear we used this somewhere, but I couldn't find it in the projects repo either. Do you know if it's actually used in a CWHQ course?

File: docs/javascript/browser-apis-and-jquery.md [hunk 4]
```diff
@@ -446,3 +446,308 @@ function increaseCatLikes() {
 
 var catIntervalID = setInterval(increaseCatLikes, 5000);
 ```
+
```
Comment (daniel-schroeder-dev) [inline docs/javascript/browser-apis-and-jquery.md:449]: I liked it that you ran prettier on your earlier commit, I'm doing that now too. Let's do that from now on. If you haven't on this commit, do it on the "fixes" commit.

File: docs/javascript/browser-apis-and-jquery.md [hunk 5]
```diff
@@ -446,3 +446,308 @@ function increaseCatLikes() {
...
+
+    <script>
+
+    function submitName(event) {
+        event.preventDefault();
+        var wizardName = $("#wizardName").val();
+        $("#heading").text(`Welcome ${wizardName}!`);
+    }
+    // add the wizardName to our heading element
```
Comment (daniel-schroeder-dev) [inline docs/javascript/browser-apis-and-jquery.md]: Again, let's be a little more explicit for what this line of code does and not what the side-effects are:

```javascript
// When the `<form>` is submitted, run the `submitName()` function.
```

File: docs/javascript/browser-apis-and-jquery.md [hunk 6]
```diff
@@ -446,3 +446,308 @@ function increaseCatLikes() {
...
+    </form>
+
+
+    <script>
+
+   function vanishInput() {
+        $("form").fadeOut();
+    }
+    // vanish form once the input value has changed
+    $("form").change(vanishInput);
```
Comment (daniel-schroeder-dev) [inline docs/javascript/browser-apis-and-jquery.md]:
```javascript
// Run the `vanishInput()` function when any `<input>` element in the `<form>` is changed.
```

File: docs/javascript/browser-apis-and-jquery.md [hunk 7]
```diff
@@ -446,3 +446,308 @@ function increaseCatLikes() {
...
+        <input id="sub" type="submit" value="Submit">
+    </form>
+
+
+    <script>
+
+    function logoChange() {
+        $("#wave").animate({ fontSize: "100px",}, "2s");
+    }
+    //change logo size once mouse is over the element
```
Comment (daniel-schroeder-dev) [inline docs/javascript/browser-apis-and-jquery.md]:
```javascript
// When the mouse is over the element with the id of `#wave`, run the `logoChange()` function.
```

File: docs/javascript/browser-apis-and-jquery.md [hunk 8]
```diff
@@ -446,3 +446,308 @@ function increaseCatLikes() {
...
+    <form>
+        <input type="text" placeholder="Enter Wizard Name" id="wizardName" autocomplete="off">
+        <input id="sub" type="submit" value="Submit">
+    </form>
+
+
+    <script>
+
+    function logoChangeTwo() {
+        $("#wave").css({color: "blue"});
```
Comment (daniel-schroeder-dev) [inline docs/javascript/browser-apis-and-jquery.md]: I'm going to try to enforce this as I build new courses, but let's go ahead and start doing it in the docs. No more POJOs as args to functions. We'll do this from now on:

```javascript
function logoChangeTwo() {
    
    var changeLogoCSS = {
        color: "blue",
    };

    $("#wave").css(changeLogoCSS);
}
```

File: docs/javascript/browser-apis-and-jquery.md [hunk 9]
```diff
@@ -446,3 +446,308 @@ function increaseCatLikes() {
...
+    }
+    //change color of logo when mouse moves
```
Comment (daniel-schroeder-dev) [inline docs/javascript/browser-apis-and-jquery.md]:
```javascript
// When the mouse moves over the element with an id of `#wave`, run the `logoChangeTwo()` function.
```

**Review by daniel-schroeder-dev — CHANGES_REQUESTED** [review]

File: docs/javascript/browser-apis-and-jquery.md (hunk 3 above)
Comment (cynthia-cwhq) [reply docs/javascript/browser-apis-and-jquery.md:502]: Found it! H14 we used hover.

**Pull request merged**